core/contacts.go

```diff
@@ -75,7 +75,7 @@ func (t *Textile) ContactUsername(id string) string {
 	if id == "" {
 		return ""
 	}
-	username = id[len(id)-7:]
+	username = id[len(id)-8:]
```

Comment:
it's 7 everywhere else... let's keep that

Reply:
oh hah, just realized what i did there... nv, reverting 😬